tests/conftest.py

```diff
@@ -697,7 +724,43 @@ def create_client(self, urgent=False, pid=None,
                 self.sync_with_hlwm()
             return w, self.winid_str(w)
 
+        def screenshot(self, win_handle):
+            geom = win_handle.get_geometry()
+            attr = win_handle.get_attributes()
+            # Xlib defines AllPlanes as: ((unsigned long)~0L)
+            all_planes = 0xffffffff
+            # Maybe, the following get_image() works differently
+            # than XGetImage(), because we still need to interpret
+            # the pixel values using the colormap, whereas the man
+            # page for XGetImage() does not mention 'colormap' at all
+            raw = win_handle.get_image(0, 0, geom.width, geom.height,
+                                       X.ZPixmap, all_planes)
+            # raw.data is a array of pixel-values, which need to
+            # be interpreted using the colormap:
+            colorDict = {}
+            for pixel in raw.data:
+                colorDict[pixel] = None
+            colorPixelList = list(colorDict)
+            colorRGBList = attr.colormap.query_colors(colorPixelList)
+            for pixelval, rgbval in zip(colorPixelList, colorRGBList):
+                # Useful debug output if something blows up again (which is likely)
+                # print(f'{pixelval} -> {rgbval.red}  {rgbval.blue} {rgbval.green}')
```

Comment:
I know this is just for debugging, but shouldn't it be red, green, blue instead of red blue green?

Could lead to some confusion while debugging.

Reply:
Indeed! Thanks, I've reordered it.

Reply:
Thanks!